terraform/modules/aws-cargohold-asset-storage/outputs.tf

```diff
@@ -26,4 +22,8 @@ output "cargohold_access_key" {
 
 output "cargohold_access_secret" {
   value = aws_iam_access_key.cargohold.secret
-}
\ No newline at end of file
+}
+
+output "talk_to_S3" {
+  value = aws_security_group.talk_to_S3.id
```

Comment:
Maybe I'm missing sth here, but I can't find the security group resource this output refers to.